workspaces/cli-shared/src/captures/avro/file-system/index.ts

```diff
@@ -0,0 +1 @@
+export const captureFileSuffix = '.optic-capture.avro';
```

Comment:
should we begin using explicit versions here? `.optic-captureV1`?

Reply:
The evolution plan is to add a new implementation of the interface that uses its own suffix, and optionally another one that composes implementations to migrate or unify older versions, depending on our requirements